src/applications/discover-your-benefits/components/BenefitCard.jsx

```diff
@@ -11,12 +12,35 @@ const BenefitCard = ({ benefit }) => {
     applyNowURL,
   } = benefit;
 
+  const handleClick = (url, text, label) => {
+    return recordEvent({
+      event: 'discover-benefits-link-click',
+      'display-text': text,
+      'link-destination': url,
+      'link-origin': label,
+    });
+  };
+
   const renderLink = (url, text, label, action = false) => {
     if (url) {
       return action === true ? (
-        <va-link-action href={url} text={text} label={label} type="secondary" />
+        <va-link-action
+          href={url}
+          text={text}
+          label={label}
+          type="secondary"
+          disableAnalytics
```

Comment:
Because this is the vanilla web component (not the React binding), this prop will likely need to be in kabab case `disable-analytics`.